test/browser/language-variants/upsert-language-variants.spec.js

```diff
@@ -0,0 +1,204 @@
+"use strict";
```

Comment:
Please remove this file. This is a compiled file and should not be here, nor should it be pushed on GH. All compiled files are put into 'dist' folder which is excluded in .gitignore.